500.html

```diff
@@ -7,19 +7,18 @@
     lastmod: 2017-08-02T17:20:30+00:00
 ---
 
-<div class="p-strip--light is-deep u-no-padding--bottom">
+<div id="main-content" class="p-strip--image is-dark vfio-hero">
```

Comment:
Please add a background-color to the class as if the image fails then its white text on white background.